src/w3c/headers.js

```diff
@@ -2,6 +2,7 @@
 // Module w3c/headers
 // Generate the headers material based on the provided configuration.
 // CONFIGURATION
+//  - level: TODO SOME EXPLANATION HERE
```

Comment:
I am still not very clear on what levels are in specs, so I am not sure how to explain it. @marcoscaceres could you suggest something? Thank you!

Reply:
heh, yeah... it's ... complicated... I don't think we need to define what a level *is* because we won't get agreement, more that if a working group wants to add a level to a specification, this is where they can do it.

Reply:
I see. XD I'd still like to add an explanation in this file to follow the convention of all config keys having some explanation. Perhaps something a bit more generic might work?  My suggestion below.
> level: the specification's level. For example, `{level: 2}` (optional)